GliaWidgets/Font.swift

```diff
@@ -3,15 +3,18 @@ import UIKit
 enum Font {
     static let headerTitle = Font.medium(20)
```

Comment:
Seems like this has to be addressed in some way. Let's check what can be done about it.

Reply:
3a549712bb2f0b808caa3a37e9f44349a5442f73